slaves/linux-cross/Dockerfile

```diff
@@ -79,6 +80,8 @@ RUN /bin/bash build_toolchain.sh arm-linux-gnueabihf
 RUN /bin/bash build_toolchain.sh mips-linux-musl
 RUN /bin/bash build_toolchain.sh mipsel-linux-musl
 
+RUN /bin/bash build_toolchain.sh armv7-linux-gnueabihf
```

Comment:
Could you add a comment here explaining why we build a whole new toolchain instead of just using the `arm-linux-gnueabihf` one above with different codegen flags? (the reason being statically linking to libstdc++ for the compiler we'll ship)